modules/redis-cluster/metadata.display.yaml

```diff
@@ -49,12 +59,25 @@ spec:
         network:
           name: network
           title: Network
+          altDefaults:
+            - type: ALTERNATE_TYPE_DC
+              value: projects/${project_id}/global/networks/default
```

Comment:
value should not have any variable? What if there is no default set for this? Or simply `default` won't work?